test/unit/spec/appWidgets/input/dynamicDropdownInputSpec.js

```diff
@@ -0,0 +1,70 @@
+/*global define*/
```

Comment:
wow, parameter control specs!!

Reply:
Not perfect, but it's a start! :)
I'm trying to not introduce any new code without a few tests. Hopefully, it'll become a trend...